salt/personalised-covers/config/srv-personalised-covers-config-config.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+return [
+	{% if pillar.elife.env in ['ci', 'dev']  %}
+	'debug' => true,
+	{% else %}
+	'debug' => false,
+	{% endif %}
+	'aws_region' => "{{ pillar.personalised_covers.aws.region }}",
+	'aws_bucket' => "{{ pillar.elife.env}}-{{pillar.personalised_covers.aws.bucket}}",
+	'elife_api' => "http://{{ pillar.elife.env }}--{{ pillar.personalised_covers.api }}",
```

Comment:
Ah, I see.  Does `builder-private` also contain the info to get correct values for dev/ci?